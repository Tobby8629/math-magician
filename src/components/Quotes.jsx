import React from 'react';

const mathquote = 'mathematics is not about numbers, equations ,computations or algorithms: is about understanding ';
const Quotes = () => (
  <div className="quotes">
    <h2>Math Quotes</h2>
    <p>
      {mathquote}
      -
      <span className="bold">william paul thurson</span>
    </p>
  </div>
);

export default Quotes;
