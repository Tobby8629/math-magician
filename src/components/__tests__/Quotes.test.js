import  renderer  from "react-test-renderer";
import Quotes from "../Quotes";

test("quotes component match snapshot", ()=>{
      const tree = renderer.create(<Quotes />).toJSON
      expect(tree).toMatchSnapshot();
}) 