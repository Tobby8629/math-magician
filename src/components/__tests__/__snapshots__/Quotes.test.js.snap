// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quotes component match snapshot 1`] = `[Function]`;
